cmd/tracee-rules/main.go

```diff
@@ -214,3 +219,31 @@ func sigHandler() chan bool {
 	}()
 	return done
 }
+
+// dropCapabilities drop all capabilities from the process
+// The function also tries to drop the capabilities bounding set, but it won't work if SYS_SETPCAP is not available.
+func dropCapabilities() error {
+	selfCaps, err := getSelfCapabilities()
+	if err != nil {
+		return err
+	}
+
+	selfCaps.Clear(capability.CAPS | capability.BOUNDS)
+	err = selfCaps.Apply(capability.CAPS | capability.BOUNDS)
+	if err != nil {
+		return fmt.Errorf("couldn't drop capabilities: %v", err)
+	}
+	return nil
+}
+
+func getSelfCapabilities() (capability.Capabilities, error) {
```

Comment:
Now that we decided to refactor during development (method suggested by @danielpacak), shouldn't we unify the "getSelfCapabilities()" functions, declared in tracee-ebpf/main.go AND tracee-rules/main.go, in a single place ?

Reply:
Maybe you should coordinate this change with https://github.com/aquasecurity/tracee/pull/1256#issuecomment-1005850610 and have your changes rebased on top of that PR after its finished. WDYT ?

Reply:
OR you can just refactor these duplicated functions and merge this PR, and ask for @danielpacak to re-check that PR. Up to you guys.

Reply:
@AlonZivony @simar7 which PR should be reviewed first? This one or #1256 ?

Reply:
> @AlonZivony @simar7 which PR should be reviewed first? This one or #1256 ?

I think @simar7 's should be first because he is introducing more generic change and it will be easy for me to rebase.on him

Reply:
👋 @AlonZivony We've merged #1256 so please go ahead with rebasing your changes.